srcpkgs/passwordsafe/template

```diff
@@ -23,8 +24,10 @@ else
 fi
 
 pre_configure() {
-	# ykpers-1 includes are hardcoded, add cross base
-	vsed -e "s|-I\(/usr/include/ykpers-1\)|-I${XBPS_CROSS_BASE}\1|" -i CMakeLists.txt
+	if [ ${CROSS_BUILD} ]; then
```

Comment:
```suggestion
	if [ -n "${CROSS_BUILD}" ]; then
```

Reply:
yes but if XBPS_CROSS_BASE is empty, `vsed` errors because nothing changed

Reply:
ok then i would suggest
```suggestion
	if [ -n "${CROSS_BUILD}" ]; then
```